wandb/sdk/internal/artifacts.py

```diff
@@ -293,24 +292,6 @@ def _cleanup_staged_entries(self) -> None:
         We need to delete them once we've uploaded the file or confirmed we
         already have a committed copy.
         """
-        staging_dir = get_staging_dir()
         for entry in self._manifest.entries.values():
-            if entry.local_path and entry.local_path.startswith(staging_dir):
-                try:
-                    os.chmod(entry.local_path, 0o600)
-                    os.remove(entry.local_path)
-                except OSError:
-                    pass
-
-
-def get_staging_dir() -> FilePathStr:
-    path = os.path.join(env.get_data_dir(), "artifacts", "staging")
-    try:
-        mkdir_exists_ok(path)
-    except OSError as e:
-        raise PermissionError(
-            f"Unable to write staging files to {path}. To fix this problem, please set "
-            f"{env.DATA_DIR} to a directory where you have the necessary write access."
-        ) from e
-
-    return FilePathStr(os.path.abspath(os.path.expanduser(path)))
+            if entry.local_path:
```

Comment:
Nit: I think all local paths are staged, but since we have this check elsewhere let's be consistent.
```suggestion
            if entry.local_path and is_staged_copy(entry.local_path):
```